Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationBroadcaster.cs

```diff
@@ -111,6 +153,20 @@ protected void OnReceive(IncomingMessage data)
                             StateSynchronizationSceneManager.Instance.ReceiveMessage(data.Endpoint, reader);
                         }
                         break;
+                    case ConnectedObserver.LocalizationMessageHeader:
```

Comment:
>LocalizationMessageHeader [](start = 43, length = 25)

I love the idea of these message headers being const values somewhere, right now the other ones are all duplicated

Reply:
Yeah, leaving as that for now.

---
In reply to: [285282753](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4460#discussion_r285282753) [](ancestors = 285282753)